### ISSUE TITLE
Make the menu button movable

### ISSUE THREAD

**Issue by notpushkin** [Lootyhoof/australium#3]
So the user can place it into the Navigation Toolbar, like it is in Firefox now.

**Comment by Lootyhoof** [comment]
This is the job of an extension, not a theme. You could try the [UI Fixer](https://addons.mozilla.org/en-US/firefox/addon/firefox-4-ui-fixer/) addon to do this. If you want something more alike the Firefox menu button, you can then add this CSS to it:

```
#fixer-menu-button > #appmenu-toolbar-button ,
#fixer-menu-button > #appmenu-button {
  border: none !important;
  background: transparent !important;
}

toolbar:not([mode="text"]) #fixer-menu-button > #appmenu-toolbar-button > .toolbarbutton-icon,
toolbar:not([mode="text"]) #fixer-menu-button > #appmenu-button > .button-box .button-icon {
  list-style-image: url("chrome://browser/skin/hamburger.png");
  margin-top: 5px;
}

#fixer-menu-button > #appmenu-button > .button-box > .button-menu-dropmarker,
#fixer-menu-button > #appmenu-toolbar-button > .button-box > .button-menu-dropmarker {
  display: none !important;
}
```

**Comment by notpushkin** [comment]
Hm, this worked, thanks!
Although I believe that this does belong to the theme. If themes can't do this, probably it could be a companion add-on instead?

**Comment by Lootyhoof** [comment]
While this could be interesting as an addon, I currently have no plans to create one. Not only would this not be able to recreate the Australis menu itself (meaning the current menu would look incorrect in the context of a toolbar button, due to assuming the menu will be opened from the top-left of the window), but implementations of this that I have seen make the button that is created basically be the appmenu-button, but movable; i.e. not a toolbar button, as I would prefer for such a feature.

In other words, while it's reasonable to expect an Australis-like theme to implement the menu button that it has, I would rather work with the native implementation than to hack in a workaround.
